stdlib/pickle.pyi

```diff
@@ -7,6 +7,11 @@ DEFAULT_PROTOCOL: int
 bytes_types: Tuple[Type[Any], ...]  # undocumented
 
 if sys.version_info >= (3, 8):
+    # Protocol 5
+    BYTEARRAY8: bytes
+    NEXT_BUFFER: bytes
+    READONLY_BUFFER: bytes
```

Comment:
Nit: It would make more sense to have these in the same place with other opcodes below. You can add a new `sys.version_info` check.

Reply:
Will do, I was not sure, which approach would be better 😄